src/OptionsRESTController.php

```diff
@@ -194,6 +194,20 @@ static function( $slug ) {
 
 		$options[ self::CUSTOMIZER_LINK ] = amp_get_customizer_url();
 
+		/**
+		 * Filters options for services to add additional REST items.
+		 *
+		 * @internal
+		 *
+		 * @param array $service_options REST Options for Services.
+		 */
+		$service_options = apply_filters( 'amp_rest_options', [] );
```

Comment:
Filters should always use extra hardening:
```suggestion
		$service_options = array_filter( (array) apply_filters( 'amp_rest_options', [] ) );
```